analytics/policy/services/business_licence_client.py

```diff
@@ -0,0 +1,60 @@
+import json
+
+import urllib3
+from models.address import Address
+from models.listing import Listing
+from policy.utils.dict_utils import merge_dicts
+from urllib3.exceptions import HTTPError
+
+
+class BusinessLicenceClient:
```

Comment:
can you explain what this client is doing, why it needs to exists and what you are trying to do with this?

Reply:
This client obj helps retrieves licence status given a licence number (from Open Data Portal)
I make it an object because there are different ways to get licence status, and we might want to get more than status from the Open Data Portal

Reply:
the reason why i am asking this question is that i want to see how this relates to a policy?
or this has nothing to do with policy? 
is it just an api client that a policy will use?

Reply:
> is it just an api client that a policy will use?
yes, the policies will check the licence status using this client

Reply:
okay i think if it's an api client to be used by policy then it's okay. imo i feel like this class is tightly coupled to the policy and it's not goign to be reusable.

Reply:
i think it's only tied to the listing? i think other modules can call this client too?